src/javascript/JContent/EditFrame/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -26,8 +26,8 @@ export const Breadcrumbs = ({nodes, setCurrentElement, onSelect}) => {
         return false;
     }, [onSelect, selection.length, setCurrentElement]);
 
-    const data = nodes.toReversed().map((n, index) => ({
-        label: n.name,
+    const data = nodes.concat(currentNode).map((n, index) => ({
```

Comment:
I am pretty sure it is already reversed as described in the ticket as part of https://jira.jahia.org/browse/BACKLOG-23413 ; so you do not need to remove the `toReversed` call

Reply:
ok